build/package/LinuxNativeInstallerDependencyVersions.targets

```diff
@@ -1,35 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="14.0" DefaultTargets="Layout" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
question, why this is removed? CLI repo should be the one producing linux packages, no?

Reply:
No, core-sdk should produce linux packages. We want CLI and toolset to not be tied to an operating system.

I removed this particular file because it referenced things I needed to change. There are more files to delete nearby.